tests/versions/apm_server.yml

```diff
@@ -1,6 +1,7 @@
 APM_SERVER:
   - '7.x'
   - '7.10'
+  - '7.10;--release'
```

Comment:
I'm not sure whether this is correct :/

Reply:
`--release` is correct, we changed it with the first release 'couse is no longer a SNAPSHOT